data-shapes-test-suite/tests/core/node/pattern-001.ttl

```diff
@@ -40,7 +40,7 @@ ex:TestShape
           rdf:type sh:ValidationResult ;
           sh:focusNode ex:Test ;
           sh:resultSeverity sh:Violation ;
-          sh:sourceConstraintComponent sh:PatternConstraintComponent ;
+          sh:sourceConstraintComponent sh:NodeKindConstraintComponent ;

```

Comment:
The original was right, this is wrong

Reply:
I have no objection with rolling this back, but I'd like the reasoning to be unambiguous from updates to the SHACL-Core document.

Reply:
I think should be `sh:PatternConstraintComponent` (which `jena-shacl` passes).

Even if it were a blank node - so some kind of argument error - it is `sh:sourceConstraintComponent` which is the constraint failing, not some kind of an error class.